static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -1,4 +1,4 @@
-// @flow strict
+// @flow
```

Comment:
☹️  😢 

> You can use Flow Strict to disallow previously-allowed unsafe patterns. This gives you improved safety guarantees that catch more bugs and make refactoring easier. And you can implement these stronger guarantees incrementally, on a file-by-file basis.

https://flow.org/en/docs/strict/

Reply:
We know @katebee , sorry! Me and @GHaberis had an extensive conversation about this and in the end had to remove strict because of how we're changing `myPageTargetting` through a callback in `getPageTargeting()`. But we've added a tech debt card to fix this post launch https://trello.com/c/TqQE4AZT/510-turn-build-page-targeting-back-to-flow-strict